### DISCUSSION TITLE
MESA Ubuntu 22.04 Installation

### DISCUSSION THREAD

**Discussion by ny2292000** [mesa/mesa#1418] [General]
I am brand new to MESA and am installing it for the first time on a Ubuntu pc.

I downloaded the mesasdk and the corresponding mesa-r22.05.1 packages.
I also installed the prerequisites
I am using Ubuntu 22.04

Output for : gfortran -v
Using built-in specs.
COLLECT_GCC=/home/mpereira/mesasdk/bin/gfortran.exec
COLLECT_LTO_WRAPPER=/home/mpereira/mesasdk/bin/../libexec/gcc/x86_64-pc-linux-gnu/12.1.0/lto-wrapper
Target: x86_64-pc-linux-gnu
Configured with: /home/user/sdk2-tmp/build/gcc/configure CC= CXX= --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu --target=x86_64-pc-linux-gnu --prefix=/home/user/sdk2-tmp/mesasdk --with-gmp=/home/user/sdk2-tmp/mesasdk --with-mpfr=/home/user/sdk2-tmp/mesasdk --with-mpc=/home/user/sdk2-tmp/mesasdk --enable-languages=c,c++,fortran --disable-multilib --disable-nls --disable-libsanitizer --enable-clocale=generic
Thread model: posix
Supported LTO compression algorithms: zlib
gcc version 12.1.0 (GCC) 

THIS IS THE OUTPUT AND ERROR:

./install

/home/mpereira/mesa-r22.05.1/const
building const package.

make: '../tester' is up to date.
export
done

mesa/const has been built, tested, and exported.

************************************************

/home/mpereira/mesa-r22.05.1/utils
building utils package.

make: '../tester' is up to date.
export
done

mesa/utils has been built, tested, and exported.

************************************************

/home/mpereira/mesa-r22.05.1/math
building math package.

make: '../tester' is up to date.
export
done

mesa/math has been built, tested, and exported.

************************************************

/home/mpereira/mesa-r22.05.1/mtx
building mtx package.

make: Nothing to be done for 'all'.

3c3
<                                                       B   -1.6215408275458779D-01    1.6391349023885564D+00   -3.8454229229650846D-02   -2.8158487838788582D+00
---
>                                                       B   -1.6215408275458812D-01    1.6391349023885562D+00   -3.8454229229650999D-02   -2.8158487838788577D+00

/home/mpereira/mesa-r22.05.1/mtx/test
TEST FAILED -- compare test_output to tmp.txt

/home/mpereira/mesa-r22.05.1/mtx
./build_and_test FAILED

Any advice would be welcomed.
Thanks,

**Comment by rht** [comment]
You meant this Mesa: https://github.com/mesa3d/mesa.

**Comment by ny2292000** [comment]
Since you mentioned this mesa3d/mesa  I went there and installed that too,
just in case.

My issue is with the latest released r22.5.1
https://github.com/MESAHub/mesa/releases/tag/r22.05.1

When I tried to install it, it didn't pass a numerical

**Reply by rht** [reply]
I meant, you likely posted in the wrong Mesa repo. There are at least 3 Mesa's: https://github.com/projectmesa/mesa, https://github.com/mesa3d/mesa, https://github.com/MESAHub/mesa. This repo (projectmesa/mesa) is meant for ABM simulation library in Python.

**Comment by ny2292000** [comment]
I watched this video about MESA installation and realized that I should have loaded the build.log file, here it goes.
[build.log](https://github.com/projectmesa/mesa/files/9637968/build.log)

**Comment by jackiekazil** [comment]
Oh dear. There are 3 mesas! 😱 I thought there were only 2. 😥

**Comment by ny2292000** [comment]
Sorry. My bad.
Thank you very much.
